content/css/concepts/typography/terms/primary & secondary type/primary & secondary type.md

```diff
@@ -0,0 +1,55 @@
+---
+Title: 'primary & secondary type'
```

Comment:
Rather than having a separate entry for this, it should go in font-family as this is a specification for that term. So, shift the information you provided here into the `font-family.md`. Lmk if you have any further questions